src/common/packager.ts

```diff
@@ -162,12 +162,15 @@ export class Packager {
 
                 let spawnOptions = { env: reactEnv };
 
+                // Since expo@37, you must specify the sourceExts parameter for the normal loading of additional files.
```

Comment:
```suggestion
                // Since expo@37, you must specify the sourceExts parameter for the normal loading of additional files:
                // (https://github.com/expo/expo-cli/blob/master/packages/xdl/src/Project.ts#L1720).
```